airflow/providers/google/cloud/hooks/dataflow.py

```diff
@@ -98,13 +98,13 @@ class DataflowJobStatus:
     Helper class with Dataflow job statuses.
     """
 
-    JOB_STATE_DONE = "JOB_STATE_DONE"
-    JOB_STATE_RUNNING = "JOB_STATE_RUNNING"
-    JOB_STATE_FAILED = "JOB_STATE_FAILED"
-    JOB_STATE_CANCELLED = "JOB_STATE_CANCELLED"
-    JOB_STATE_PENDING = "JOB_STATE_PENDING"
-    FAILED_END_STATES = {JOB_STATE_FAILED, JOB_STATE_CANCELLED}
-    SUCCEEDED_END_STATES = {JOB_STATE_DONE}
+    DONE = "DONE"
```

Comment:
Do you still need to keep the values in the `JOB_STATE_DONE` format to match Dataflow returned statuses?

Reply:
I agree. If the API uses a prefix, we can keep it on our side aalso.

https://cloud.google.com/dataflow/docs/reference/rest/v1b3/projects.jobs#Job.JobState